cmd/server-load-simulator/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# build stage
```

Comment:
a YAML like this would work?

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: server-load-simulator
  namespace: default
spec:
  replicas: 1
  selector:
    matchLabels:
      role: server-load-simulator
  template:
    metadata:
      labels:
        role: server-load-simulator
    spec:
      containers:
      - args:
        - "-maxValue=100"
        - "-jitter=0.15"
        image: docker.io/dgkanatsios/thundernetes-server-load-simulator:3fa3c4e
        imagePullPolicy: IfNotPresent        
        name: simulator
        ports:
        - containerPort: 8080
        resources:
          limits:
            cpu: 50m
            memory: 100Mi
          requests:
            cpu: 50m
            memory: 100Mi
        securityContext:
          allowPrivilegeEscalation: false
---
apiVersion: v1
kind: Service
metadata:
  name: server-load-simulator
  namespace: default
spec:
  ports:
  - port: 8080
    protocol: TCP
    targetPort: 8080
  selector:
    role: server-load-simulator
  type: ClusterIP
```

Reply:
output looks good!
```
dgkanatsios@desktopdigkanat:~$ k logs server-load-simulator-649799c8b-mrnxq
time="2022-07-18T23:25:42Z" level=info msg="Starting server-load-simulator"
time="2022-07-18T23:25:42Z" level=info msg="Config:config{100 1h0m0s 0.15 8080}"
time="2022-07-18T23:25:52Z" level=info msg="Simulated instances:value56"
time="2022-07-18T23:26:02Z" level=info msg="Simulated instances:value47"
time="2022-07-18T23:26:12Z" level=info msg="Simulated instances:value50"
time="2022-07-18T23:26:22Z" level=info msg="Simulated instances:value55"
time="2022-07-18T23:26:32Z" level=info msg="Simulated instances:value54"
time="2022-07-18T23:26:42Z" level=info msg="Simulated instances:value60"
time="2022-07-18T23:26:52Z" level=info msg="Simulated instances:value55"
time="2022-07-18T23:27:02Z" level=info msg="Simulated instances:value55"
time="2022-07-18T23:27:12Z" level=info msg="Simulated instances:value56"
time="2022-07-18T23:27:22Z" level=info msg="Simulated instances:value53"
time="2022-07-18T23:27:32Z" level=info msg="Simulated instances:value58"
time="2022-07-18T23:27:42Z" level=info msg="Simulated instances:value56"
```

Reply:
![image](https://user-images.githubusercontent.com/8256138/179814495-38e3d9eb-676b-42bb-aa67-84e5a6d301c7.png)